src/cast_core/src/MonotonicLoader.cpp

```diff
@@ -95,6 +100,8 @@ void genny::actor::MonotonicLoader::run() {
                             auto tmpDoc = config->documentExpr();
                             auto builder = bsoncxx::builder::stream::document();
                             builder << "_id" << ++id_num;
+                            builder << config->fieldIncreasingByOffsetFromID 
+                                << (id_num + config->offsetFromID);
```

Comment:
Note that the changes in this file are causing failures for other workloads. These fields should be optional, since other workloads do not require this functionality.